src/anyio/_backends/_asyncio.py

```diff
@@ -499,28 +499,25 @@ async def __aexit__(
         while self.cancel_scope._tasks:
             try:
                 await asyncio.wait(self.cancel_scope._tasks)
-            except asyncio.CancelledError:
+            except asyncio.CancelledError as exc:
```

Comment:
`CancelledError` is already imported and using it differently here made me think it was a different type

```suggestion
            except CancelledError as exc:
```